app/models/ncr/organization.rb

```diff
@@ -23,5 +23,9 @@ def ool?
     def whsc?
       code == WHSC_CODE
     end
+
+    def ba_6x_tier1_team?
```

Comment:
done in 44b9339

I'd rather leave the explicit cases in the `spec/services/ncr/approval_manager_spec.rb` so that it is clear that it does not affect `BA80` expense types and applies equally to `BA60` and `BA61` (since the approver only has `BA61` in the name, per the trello card).